src/components/chip/chip.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import "./chip.scss"
+
+const Chip = () => {
```

Comment:
Suggestion: pass props  here.

Reply:
Should I deconstruct as { text } for the props being passed, or just go ahead and pass props itself?